src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -200,6 +213,25 @@ def _wrap_modules(self, layer, config):
         raise NotImplementedError()
 
 
+    def add_activation_collector(self, collector):
```

Comment:
a little confused about this function, you define it for users to use it, but in your implementation activation pruners do not use it...

Reply:
Updated.